### ISSUE TITLE
CSO files are not working on PSP

### ISSUE THREAD

**Issue by Danixu** [unknownbrackets/maxcso#64]
Hello,

I have always used your program to compress all the ISO files I have to CSO and save space on my PSP and always was working perfectly.
Yesterday I have decided to use my PSP again and I have proceed to compress my library to CSO using the latest version, but I have noticed that almost all the games were not working.

I have reinstalled the firmware several times trying different CFW tinking that maybe was a firmware problem, but I got the same behaviour. My last test was to dump a clean ISO from an UMD I have, and that ISO was working perfectly. Compressing that ISO to CSO made it to fail again, so I have decided to test the UMDGen CSO compression, which is working perfectly.

I am using the default options to generate the CSO:

```
maxcso "Pusuit Force.iso"
```

The only difference between both images I have noticed, is that MaxCSO image is a few kb smaller. I have tested older versions, and the version v1.12.0 seems to be working (also is a few kb smaller than the UMDGen, but bigger than latest MaxCSO version).

Best regards.

**Comment by jcchikikomori** [comment]
Hi @Danixu
Have you tried using the -o (output file) parameter?

```
maxcso "Pusuit Force.iso" -o "Pusuit Force.cso"
```

**Comment by Danixu** [comment]
Hello,

Is there any difference?, by default the program creates a cso file.

Best regards.

**Comment by unknownbrackets** [comment]
[v1.13.0](https://github.com/unknownbrackets/maxcso/releases/tag/v1.13.0) added libdeflate compression, which somehow makes PSP decompression crash.

You can use `--no-libdeflate` to prevent libdeflate from being used even in v1.13.0, but I disabled it again in the latest git.  Haven't released a new stable version yet, see 161f99d81f81716943069387bb0234ce5a1355b3.

-[Unknown]

**Comment by Danixu** [comment]
Thanks!!.
